core/App/navigators/RootStack.tsx

```diff
@@ -187,7 +188,7 @@ const RootStack: React.FC<RootStackProps> = (props: RootStackProps) => {
     return (
       <Stack.Navigator initialRouteName={Screens.Splash} screenOptions={{ ...defaultStackOptions, headerShown: false }}>
         <Stack.Screen name={Stacks.TabStack} component={TabStack} />
-        <Stack.Screen name={Stacks.ConnectStack} component={ConnectStack} options={{ presentation: 'modal' }} />
```

Comment:
Still learning but I believe this is the preferred way. We don't wrap screens in `<Modal>...</Modal>` any longer but rather tell the navigator how we want to to preset the screen (modal or otherwise). The exception being in-line modals that, for lack of a better term, are a child of some other screen.